README.md

```diff
@@ -188,84 +182,53 @@ and install the dependencies:
 cd bandada && yarn
 ```
 
-## 📜 Usage
-
-### Copy the API environment variables
+## 🔧 Configuration
 
-Run this command to copy the [environment variables used by the API](https://github.com/privacy-scaling-explorations/bandada?tab=readme-ov-file#environment-variables):
-
-```bash
-cp apps/api/.env.example apps/api/.env
-```
+### Prerequisities
```

Comment:
What about adding the prerequisites before the Installation so that devs can use the right config from the beginning?

Reply:
Yes, I have moved the whole `configuration` section before the `installation` one.